src/Pages/ForgotPassword.cshtml.cs

```diff
@@ -36,10 +36,10 @@ public class ForgotPasswordModel : PageModel
 
         private EmailSender EmailSender { get; }
         private UserDatabase UserDatabase { get; }
-        public ForgotPasswordModel(EmailSender emailSender, UserDatabase userDatabase)
+        public ForgotPasswordModel(UserDatabase userDatabase, EmailSender emailSender)
```

Comment:
Just interested, why was this changed?

Reply:
No reason really I think, just being ocd about pointless things :D